modules/installation-aws-config-yaml.adoc

```diff
@@ -87,35 +88,38 @@ platform:
       adminContact: jdoe
       costCenter: 7536
 ifdef::vpc[]
-    subnets: <6>
+    subnets: <7>
     - subnet-1
     - subnet-2
     - subnet-3
 endif::vpc[]
 pullSecret: '{"auths": ...}' <1>
 ifdef::vpc[]
-fips: false <7>
-sshKey: ssh-ed25519 AAAA... <8>
+fips: false <8>
+sshKey: ssh-ed25519 AAAA... <9>
 endif::vpc[]
 ifndef::vpc[]
-fips: false <6>
-sshKey: ssh-ed25519 AAAA... <7>
+fips: false <7>
+sshKey: ssh-ed25519 AAAA... <8>
 endif::vpc[]
 ifdef::private[]
-publish: Internal <9>
+publish: Internal <10>
 endif::private[]
 ----
 <1> Required. The installation program prompts you for this value.
-<2> If you do not provide these parameters and values, the installation program
+<2> Optional. Add this parameter to change how the Cloud Credential Operator
```

Comment:
Yes, this is optional. CCO will try to determine mint or passthrough mode dynamically if this is blank. But this is absolutely required for AWS when a user wants to skip permissions checking (we may have separate docs on that?... i don't remember).

Maybe it is clearer like this:
<2> Optional. Add this parameter to force the Cloud Credentials Operator (CCO) into the specified mode (instead of having CCO dynamically try to determine the credential's capabilities).